keyboards/westfoxtrot/cypher/readme.md

```diff
@@ -0,0 +1,15 @@
+# cypher
+
+![cypher](https://github.com/westfoxtrot/Cypher_PCB)
+
+PCB for the cypher keyboard
+
+Keyboard Maintainer: [westfoxtrot](https://github.com/westfoxtrot)
```

Comment:
Lines 7 and 8 need to end in two spaces for GitHub to render this file properly.

```suggestion
Keyboard Maintainer: [westfoxtrot](https://github.com/westfoxtrot)  
```